routing_model/index.html

```diff
@@ -714,12 +714,34 @@ <h2>Process Adult/Elder promotion and demotion needed from last check</h2>
       LoopEnd --> LoopStart
   </div>
 
-  <h2>ProcessMerge Sub-routine</h2>
+  <h2>Handling merges</h2>
   <div class=description>
   <p>Vote for ParsecOurMerge, and take over handling any ParsecNeighbourMerge.<br/>
      Complete when one merge has completed, and a NewSectionInfo is consensused.<br/>
-     If Multi stage merges are required, they will require calling this function again<br/>
+     If multi-stage merges are required, they will require calling this function again<br/>
+     While in this sanctuary, our SectionInfo shall not be disturbed by elder/adult changes.
+     This avoids troublesome situations where different nodes may send different SectionInfo's to the neighbour section and cause confusion.
```

Comment:
No, we are in shared state, so this cannot happen.
What could happen, is we generate a new SectionInfo after the one we said would be the parent of the merged SectionInfo.

So maybe:
"This stops us from changing our SectionInfo after indicating to our neighbour the last SectionInfo before merge"